obspy/io/gse2/libgse2.py

```diff
@@ -465,13 +465,33 @@ def parse_sta2(line):
     """
     header = {}
     try:
+        lat = line[15:24].strip()
```

Comment:
But maybe you specifically want it to fail for something like `line[15:24] == 'abcdef'`?